Dockerfile

```diff
@@ -1,20 +1,39 @@
-FROM eclipse-temurin:11.0.25_9-jdk-noble
+FROM eclipse-temurin:11.0.25_9-jdk-noble as build
 # Ubuntu 24.04 LTS noble ^^
 
+RUN apt update && apt install -y git
+
+WORKDIR /tmp/kbsdk
+
+# dependencies take a while to D/L, so D/L & cache before the build so code changes don't cause
+# a new D/L
+# can't glob *gradle because of the .gradle dir
+COPY build.gradle gradlew settings.gradle /tmp/kbsdk
+COPY gradle/ /tmp/kbsdk/gradle/
+RUN ./gradlew dependencies
+
+# Now build the code
+# for the git commit
+COPY .git /tmp/kbsdk/.git/
+COPY src /tmp/kbsdk/src/
+RUN ./gradlew prepareRunnableDir
+
+
+FROM eclipse-temurin:11.0.25_9-jdk-noble
+
+# install docker
+# spent too much time trying to d/l in the build step and install the debs here, not worth it
```

Comment:
which debs were these that you were trying to download?

(or, there's a minor typo in that comment)

Reply:
The docker *.deb files

Reply:
These ones: https://github.com/kbase/kb_sdk_plus/pull/75/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557R39-R43